.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "tasks/computer-vision/image-generation/text-to-image/AttnGAN"]
+	path = tasks/computer-vision/image-generation/text-to-image/AttnGAN
+	url = ssh://git@github.com/mlaradji/AttnGAN.git
```

Comment:
Would it be possible to switch the url to https as it might failed for consumers who might no have ssh keys added in their github account. Or might not even have github account.

Reply:
Good points. I've made the change.